src/com/aniket/sorting_algorithms/BubbleSort.java

```diff
@@ -19,7 +19,7 @@
 	}
 	
 	
-	/*optimized -- now instead of going to the n-1 position every time we will ignore 
+	/* Better optimized -- now instead of going to the n-1 position every time we will ignore 
```

Comment:
Can be reversed as : 
```suggestion
	/* Best optimized -- now instead of going to the n-1 position every time we will ignore 
```